README.md

```diff
@@ -335,3 +335,74 @@ The response contains the same data as when the repair was created:
 
 As above: if the repair was created without work orders, an empty array of
 workOrders should be returned (rather than no key or a null value)
+
+
+## Get Appointments for Work Order
+
+Returns a list of appointments matching the given criteria.
```

Comment:
Can we be more specific? We're returning appointments for a specific work order, right?